projects/ngx-advanced-img/src/lib/classes/bitmap.ts

```diff
@@ -1,5 +1,6 @@
 import * as exif from 'exifr';
 import mime from 'mime';
+import heic2any from 'heic2any';
```

Comment:
We should add this package as a peer dependency in the project package mainfest.